sdk/deviceupdate/azure-iot-deviceupdate/CHANGELOG.md

```diff
@@ -1,4 +1,20 @@
 # Release History
 
+## 1.0.0b2 (2022-01-19)
+
+### Features Added
+    
+  - Added DeviceManagementOperations
+  - Added DeviceUpdateOperations
+  - Added operation `send_request` to send customized http request conveniently
+
+### Breaking Changes
+
+  - Removed all models
+  - Removed DeploymentsOperations
+  - Removed UpdateOperations
+  - Dropped support for Python2.7(https://github.com/Azure/azure-sdk-for-python#disclaimer)
+  - parameter `account_endpoint` of DeviceUpdateClient is renamed to `endpoint`
```

Comment:
also @msyyc I think it might be hard to have the changelog tool know what is being renamed into what. Since you're already adding manul entries to the changelog, can you switch the "removal and addition" changelog into one breaking change entry that lists it as a rename instead? thanks!